bucoffea/monojet/definitions.py

```diff
@@ -560,7 +562,7 @@ def monojet_regions(cfg):
 
     # additional regions to test out deep ak8 WvsQCD tagger
     for region in ['sr_v','cr_2m_v','cr_1m_v','cr_2e_v','cr_1e_v','cr_g_v']:
-        for wp in ['loose', 'tight']:
+        for wp in ['loose', 'medium', 'tight', 'inclusive']:
```

Comment:
Suggest do make the additional regions ('medium' and 'inclusive') dependent on a config option. Always running them alongside the normal mono-V analysis uses a lot of resources for no reason.

Reply:
Done. Now the 'medium' and 'inclusive' WPs are only run when doing the mistag studies